README.md

```diff
@@ -1,3 +1,17 @@
+# Note: forked
+
+This resource has been forked to add support for [retrieving AWS credentials
+from IAM instance profiles][IAM]. [Docker Hub][hub] builds a container from
+the `gds` branch. There is no automated testing because the integration
+tests require credentials, so you will need to run the tests yourself.
+
```

Comment:
Good point - updated. When this is merged I'll update the other PR with the same content.